.bluemix/pipeline.yml

```diff
@@ -82,7 +85,7 @@ stages:
       fi
       
       # Tracking Metrics
-      if [ "YES" != "$DISABLE_METRIC" ]; then
+      if [ "NO" != "$DISABLE_METRIC" ]; then
       echo "============================================================="
```

Comment:
i just realized this block is not indented.

Reply:
Sorry I didn't indent this block because the [EOF](https://github.com/Tomcli/chatbot-deployer/blob/tracker/.bluemix/pipeline.yml#L108) portion of the code doesn't work if there are spaces after it. I will try to find if there's another solution.